tests/test_backend.py

```diff
@@ -171,14 +171,16 @@ def test_client_get_many(self, get_cluster_info):
             ret = backend.get_many(['key1', 'key2', 'key3'])
             assert ret == {'key2': 1509111630.048594}
 
+    @patch('pymemcache.client.base.Client.set_many')
     @patch.object(ConfigurationEndpointClient, 'get_cluster_info')
-    def test_client_set_many(self, get_cluster_info):
+    def test_client_set_many(self, get_cluster_info, set_many):
         from django_elastipymemcache.backend import ElastiPymemcache
 
         servers = ['h1:0', 'h2:0']
         get_cluster_info.return_value = {
             'nodes': servers
         }
+        set_many.side_effect = [[':1:key1'], [':1:key2']]
```

Comment:
Changes around [here](https://github.com/pinterest/pymemcache/compare/64109c2b140807d19cb65ce8f34ff0121b468fe3...d12d59de5ab91335320748bca5e34a3e6d836ec8#diff-21278e9eef4559b8da3fee201fae657fba60b1612d458aa4efaf673b913ede35R341-R346) at v3.4.4 made an unexpected return value mutation.
IMO, It seems a bug. Unfortunately, those implementation doesn't fix yet.

So, I apply a mock to the set_many method to perform returning dummy values as if failed to set those keys.